src/DssSpell.t.base.sol

```diff
@@ -323,7 +323,7 @@ contract DssSpellTestBase is DSTest, DSMath {
             osm_mom_authority:     address(chief),          // OsmMom authority
             flipper_mom_authority: address(chief),          // FlipperMom authority
             clipper_mom_authority: address(chief),          // ClipperMom authority
-            ilk_count:             45                       // Num expected in system
+            ilk_count:             47                       // Num expected in system
         });
 
```

Comment:
I think we should also set `vow_hump_max` (L316) to 90M in system values after spell.

Reply:
ahh good catch